### DISCUSSION TITLE
Port to solid-js

### DISCUSSION THREAD

**Discussion by uma-uc** [TanStack/table#3697] [General]
I am attempting to port to solid-js. To that end, does anyone have any pointers. 

Specifically, I could not find _useTable, useFilters, useGlobalFilter, useAsyncDebounce, useSortBy, usePagination_ (amongst others) definitions in src.

I suppose it gets generated but it is unclear as how it is being done. Please excuse my ignorance

**Comment by tannerlinsley** [comment]
Are you referring to v8 or the stable v7?

Tanner Linsley

**Reply by uma-uc** [reply]
Sorry V8 (I should have clarified) since it is TS

**Reply by tannerlinsley** [reply]
V8 doesn’t have a plug-in system. The core includes all of the basic architectural features of the previous plug-in system. You can still override un invert control of those features through options similar to the plug-in system, but without the Typescript headaches set a plug-in system brings

**Reply by uma-uc** [reply]
ok. how do I run the examples locally? When I do so I get the error Module not found: Can't resolve './createTable' (I tried running in examples/basic folder
